source/_reference-g-s.html.md.erb

```diff
@@ -3302,7 +3304,7 @@ Searches all sheets that the user can access, for the specified text.
 -----|-----|
 <%= HEADERS %> | <%= HEADER_AUTH %>
 <%= PARAMETERS %> | **query** (required): text with which to perform the search. Enclose in double-quotes for an exact search.
- | **include** (optional): when specified with a value of **favoriteFlag**, response indicates which returned items are favorites:<ul><li>**favorite** -- folders, reports, sheets, Sights, templates, and workspaces will have the property **favorite: true**</li><li>**parentObjectFavorite** -- attachments, discussions, profile fields, and rows will have the property **parentObjectFavorite: true**</li></ul>
+ | **include** (optional): when specified with a value of **favoriteFlag**, response indicates which returned items are favorites:<ul><li>**favorite** -- folders, reports, sheets, Sights, templates, and workspaces will have the property **favorite: true**</li><li>**parentObjectFavorite** -- attachments, discussions, summary fields, and rows will have the property **parentObjectFavorite: true**</li></ul>
  | **location** (optional): when specified with a value of **personalWorkspace**, limits the response to only those items in the user's workspaces.
  | <%= PARAMETER_MODIFIEDSINCE %>
  | **scopes** (optional): If search fails, try using an array for each type of this comma-separated list of search filters:<ul><li><b>attachments</b></li><li><b>cellData</b></li><li><b>comments</b></li><li><b>folderNames</b></li><li><b>profileFields</b></li><li><b>reportNames</b></li><li><b>sheetNames</b></li><li><b>sightNames</b></li><li><b>templateNames</b></li><li><b>workspaceNames</b></li></ul>
```

Comment:
Still have a `profileFields` here

Reply:
@TaranChristensen it appears that the actual scope used is profileFields. I.e. a scope of summaryFields returns an unrestricted search. Is this just a bug?

Reply:
Uh oh! This sounds like a bug. I'll definitely investigate and, if a bug, attempt to rectify it in r84. The idea, according to Marketing, is that we can't have `profile` in the public-facing API

Reply:
Update on this: The API will be using `summaryFields` for `searchScope` as of r84.. FWIW, in testing I found that if someone uses a search scope that doesn't exist, we merely ignore it.

Reply:
Since this is getting fixed in r84, I'm going to leave "summary fields" in the doc.

Reply:
@kfansler Kevin, we still have a profileFields in the search scope though that needs to be changed to summaryFields.